### PR TITLE
Update TBs needed for full history storage

### PR DESCRIPTION
**Pull request by jhaaaa**
Per https://ripplelabs.atlassian.net/browse/DOC-1603

### CODE REVIEW

**Comment by jhaaaa** [comment]
Hi @mtrippled! When you have a moment - would you mind taking a look at this PR that addresses your ticket https://ripplelabs.atlassian.net/browse/DOC-1603? 

The TB value is from 5/30 - perhaps there is a even more recent value we want to provide?

Thank you!

File: content/tutorials/manage-the-rippled-server/capacity-planning.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ path={path_to_ledger_store}
 
 ### Historical Data
 
-The amount of historical data that a `rippled` server keeps online is a major contributor to required storage space. At the time of writing (2018-03-01), a `rippled` server stores about 12GB of data per day. You can expect this amount to grow as transaction volume increases across the XRP Ledger network. You can control how much data you keep with the `online_delete` and `advisory_delete` fields.
+The amount of historical data that a `rippled` server keeps online is a major contributor to required storage space. At the time of writing (2018-05-30), a `rippled` server stores about 12GB of data per day and requires 7.9TB to store the full history of the XRP Ledger. You can expect this amount to grow as transaction volume increases across the XRP Ledger network. You can control how much data you keep with the `online_delete` and `advisory_delete` fields.
```
Comment (mDuo13) [inline content/tutorials/manage-the-rippled-server/capacity-planning.md]: Per the JIRA ticket, it's up to 8.4 TB as of 2018-10-29

Comment (jhaaaa) [reply content/tutorials/manage-the-rippled-server/capacity-planning.md]: @mDuo13 - Thank you for keeping an eye out! Yes, saw the comment from Mark 👍 , just hadn't had time to update the PR. Updated now! 🎊 Would you mind taking a look, please?

**Comment by jhaaaa** [comment]
@mtrippled - Thank you for updating the ticket with the current full history size 🙌 . I've updated the PR.

**Review by mDuo13 — APPROVED** [review]
:+1:

**Pull request merged**